app/uk/gov/hmrc/selfservicetimetopay/config/frontendWiring.scala

```diff
@@ -80,6 +84,13 @@ object EligibilityConnector extends EligibilityConnector with ServicesConfig {
   val http = WSHttp
 }
 
+trait TimeToPayController extends FrontendController with Actions {
+  override lazy val authConnector: AuthConnector = FrontendAuthConnector
+  lazy val sessionCache: KeystoreConnector = SessionCacheConnector
+
+  protected lazy val Action = CheckSessionAction andThen PlayAction
```

Comment:
Override Action to do a pre-check of the session for the presence of a sessionId. Redirects to start of flow if not present and creates one.